scripts/build-compiler.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# To be run from the root of the Flambda backend repo
+
+git clean -Xdf
+autoconf
+./configure --enable-ocamltest --enable-warn-error --enable-dev --prefix=`pwd`/_install
+make install
```

Comment:
I would probably "inline" this code
into the `profile-compiler-build.sh`
script - possibly as a function.